store/registry_test.go

```diff
@@ -176,19 +176,10 @@ func Test_registryStore_FindScannerShardIDForBot(t *testing.T) {
 			tt.name, func(t *testing.T) {
 				rs, _ := NewRegistryStoreFromClients(context.Background(), tt.fields.cfg, tt.fields.mc(t), tt.fields.rc(t))
 				gotShardID, gotShards, gotTarget, err := rs.FindScannerShardIDForBot(tt.args.agentID, tt.args.scannerAddress)
-				if (err != nil) != tt.wantErr {
-					t.Errorf("FindScannerShardIDForBot() error = %v, wantErr %v", err, tt.wantErr)
-					return
-				}
-				if gotShardID != tt.wantShardID {
-					t.Errorf("FindScannerShardIDForBot() gotShardID = %v, want %v", gotShardID, tt.wantShardID)
-				}
-				if gotShards != tt.wantShards {
-					t.Errorf("FindScannerShardIDForBot() gotShards = %v, want %v", gotShards, tt.wantShards)
-				}
-				if gotTarget != tt.wantTarget {
-					t.Errorf("FindScannerShardIDForBot() gotTarget = %v, want %v", gotTarget, tt.wantTarget)
-				}
+				assert.Equal(t, err != nil, tt.wantErr)
+				assert.Equal(t, gotShardID, tt.wantShardID)
```

Comment:
small nit, i think the expected is supposed to be the first argument (otherwise the error message will be flipped, like expected [gotShardID] but got [watShardID])